modules/netns-isolation.nix

```diff
@@ -53,12 +53,11 @@ in {
       type = types.attrsOf (types.submodule {
         options = {
           id = mkOption {
-            # TODO: Exclude 10
             # TODO: Assert uniqueness
-            type = types.int;
+            type = types.ints.between 10 255;
```

Comment:
This includes 10 and 255, shouldn't it be `types.ints.between 11 255;`?

Reply:
Hah, that's what reviews are for :cat:. Thanks for catching this.